pos_order_to_sale_order/models/pos_config.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    iface_create_draft_sale_order = fields.Boolean(
```

Comment:
Just a remark: it could have been interesting to have a checkbox "Create Sales Order" and a selection box with all possible states (you give 3 but multiple others could be imaginable).

Reply:
(Saw your previous message later :tongue: anyway both usability options have pros and cons :smile: )